hoot-core-test/src/test/cpp/hoot/core/io/ObjectOutputStreamTest.cpp

```diff
@@ -83,7 +83,7 @@ class ObjectOutputStreamTest : public CppUnit::TestFixture
     uut.writeObject(t);
     uut.flush();
 
-    //_dumpString(ss.str());
+    //TestUtils::dumpString(ss.str());
```

Comment:
Even though this is in a comment, the `_dumpString` function was written two or three times in different files so I put it into the `TestUtils` class and updated all usages.